pandas/_libs/tslibs/period.pyx

```diff
@@ -290,6 +317,20 @@ cdef int dInfoCalc_SetFromAbsDateTime(date_info *dinfo,
     """
     Set the instance's value using the given date and time.
     Assumes GREGORIAN_CALENDAR.
+
+    Parameters
+    ----------
+    dinfo : date_info*
```

Comment:
pointer to

Reply:
That's what the asterisk is for (this notation is used in docstrings elsewhere).  Have a preferred syntax?

Reply:
ok, you are removing this anyhow IIRC? (in follow on)

Reply:
Yes.